description.md

```diff
@@ -20,6 +19,43 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+The primary goal of drills is to promote mastery and internalisation of skills or knowledge.
+By repeating tasks or exercises, students can reinforce their understanding and enhance their ability to apply learned concepts effectively.
+However, it's important to balance drills with other types of learning activities to maintain student engagement and prevent monotony.
+For example, drills can be combined with homework assignments, group discussions, and other activities to provide a more varied learning experience.
+The difference is that while drills are repetitive, simple, and focused on a single skill or concept, homework activities can be more complex and open-ended.
+
+The effectiveness of drills in education depends on various factors, including the type of drill, the subject matter, and the student's learning style.
+
+Some keys considerations when designing drills include:
+
+- **Skill acquisition**: By providing repetitive practice, drills help students develop accuracy and speed.
+- **Reinforcement**: Drills should be used to reinforce skills and knowledge that have already been taught.
+- **Building foundations**:  By practicing fundamental skills and concepts repeatedly, students can develop a solid base of knowledge, which can support their learning in more complex areas.
+- **Confidence**: Drills can boost student confidence as they see their progress through repeated practice.
+- **Engagement**: Start with simple drills and gradually increase the difficulty level to prevent students from becoming overwhelmed.
+
+To overcome this limitation, some key considerations include:
+
+- **Variety**: Drills should be varied to prevent monotony and keep students engaged.
+- **Feedback**: Students should receive feedback on their performance to help them identify areas for improvement.
+- **Critical thinking**: Some open-ended questions can be included in drills to encourage students to think critically about the material.
+
+There are several ways to organize these drills:
+
+- **Individual drills**: Students complete drills individually, either in class or at home.
+- **Group drills**: Students work in groups to complete drills.
+- **Competitive drills**: Students compete against each other to complete drills.
+- **Timed drills**: Students complete drills within a specified time limit.
+
+Infrastructures for drills:
+
+- **[Google Forms](https://www.google.com/forms/about/)**: Online tool that allows you to create surveys, quizzes, and other types of forms.
+- **[Kahoot](https://kahoot.com/)**: Online tool that allows you to create quizzes and other types of interactive learning activities.
+- **[Moodle](https://moodle.org/)**: Open-source learning management system that allows you to create quizzes and other types of learning activities.
```

Comment:
These are for quizzes, not for drills. Drills are hands-on and / or open-ended activities.